djstripe/models.py

```diff
@@ -26,6 +27,7 @@
 from .signals import subscription_made, cancelled, card_changed
 from .signals import webhook_processing_error
 
+_log = logging.getLogger(__name__)
```

Comment:
That's cool, I respect your choice but I have to ask why as there's no chance anyone will go back later and put them in and with a tedious thing like payments and subscriptions, logging is very helpful. Again, I'm cool either way, just asking :)